packages/ecc-client-ga4gh-tes/src/components/tes-create-run/definition/tesCreateRun.template.ts

```diff
@@ -0,0 +1,408 @@
+import { html, repeat, when } from '@microsoft/fast-element';
+import {
+  provideFASTDesignSystem,
+  fastTextField,
+  fastSwitch,
+  fastButton,
+  fastDivider,
+} from '@microsoft/fast-components';
+import TESCreateRun from './tesCreateRun.js';
+
+provideFASTDesignSystem().register(
+  fastTextField(),
+  fastSwitch(),
+  fastButton(),
+  fastDivider()
+);
+
+const ExecutorsTemplate = html<TESCreateRun>`
+  ${repeat(
+    (x) => x.taskExecutors,
+    html`
+      <div class="executors">
+        <div class="label-input commands">
+          <label for="command">Command:</label>
+          <fast-text-field
+            type="text"
+            id="command"
+            name="command"
+            class="input"
+            :value=${(x) => x.command.join(',')}
+            @input=${(x, c) =>
+              c.parent.handleExecutorsCommandChange(c.event, x)}
+          ></fast-text-field>
+        </div>
+        <div class="label-input image">
+          <label for="command">Image:</label>
+          <fast-text-field
+            type="text"
+            id="image"
+            name="image"
+            class="input"
+            :value=${(x) => x.image}
+            @input=${(x, c) => c.parent.handleExecutorsImageChange(c.event, x)}
+          ></fast-text-field>
+        </div>
+        <div class="container env-container">
+          <span class="data-button">
+            <fast-button
+              class="add"
+              id="add-env"
+              @click=${(x, c) => c.parent.addEnv(x)}
+            >
+              Add Env
+            </fast-button>
+          </span>
+          <div class="env">
+            ${repeat(
+              (x) => Array.from(Array(Object.keys(x.env).length).keys()),
```

Comment:
this is a bit of a readability issue imo, ideally we don't want to be writing code like this. Maybe turn this into a proper function and split the logic into multiple lines

Reply:
Linter is keeping it in one line, this is used in just one place so making a separate function doesn't seem right. Should I still make a separate func.

Reply:
i mean like you can do something a little more like this. 
```
() => {
    write your logic here
    return here
}
```
instead of a one liner

Reply:
i mean something like this. Ideally we also want to avoid using objects where mapping might be an issue. Some frameworks are worse than others when handling this but let's try to avoid this altogether.

```
(x) => {
      const envKeys = Object.keys(x.env);
      const { length } = envKeys;
       const envArr = Array.from(Array(length).keys());

        return envArr;
  }
```